util/src/main/java/org/hyperledger/besu/util/Slf4jLambdaHelper.java

```diff
@@ -44,7 +44,7 @@ public static void infoLambda(
 
   public static void debugLambda(
       final Logger log, final String message, final Supplier<?>... params) {
-    if (log.isDebugEnabled()) {
+    if (log.isInfoEnabled()) {
```

Comment:
My understanding is that all the log changed from debug to info are for surfacing some more logs needed during the development.
I would like to suggest a different approach, that avoid to make all these changes, that also makes harder to focus on actual changes.
My suggestion is to revert the changes to the log level, and instead use a custom log4j configuration to enable debug level only on the classes you are interested in during the development. This way the change list in the PR will be much smaller, and there is no effort in reverting these changes before merging to main, and avoid the risk of missing to revert some.
I am available to help setting the custom log4j conf, in case you need.

Reply:
Seconded! You can set an environment var much like:  

`LOG4J_CONFIGURATION_FILE=/Users/jflo/src/besu_log4j.xml`

And provide a log4j config xml and set the logging level per class or package as you need to.